packages/server/lib/plugins/simple_tsify.js

```diff
@@ -0,0 +1,39 @@
+let through = require('through2')
+
+const isJson = (code) => {
+  try {
+    JSON.parse(code)
+  } catch (e) {
+    return false
+  }
+
+  return true
+}
+
+module.exports = function (b, opts) {
+  const chunks = []
+
+  return through(
+    (buf, enc, next) => {
+      chunks.push(buf.toString())
+      next()
+    },
+    function (next) {
+      const ts = opts.typescript
+      const text = chunks.join('')
+
+      if (isJson(text)) {
+        this.push(text)
+      } else {
+        this.push(ts.transpileModule(text, {
+          compilerOptions: {
```

Comment:
i kind of wonder about this approach, no objection and i love that it "just works" but i guess maybe to clarify: we will definitely ignore whatever people have in their own `tsconfig.json`

Reply:
(that seems reasonable honestly, just thinking about it)

Reply:
Yes, it does. Because for the speed, it ignores the type check. If we do type check, it takes about 10 sec to load the test.